### DISCUSSION TITLE
[Feature] Timeline in album

### DISCUSSION THREAD

**Discussion by LovelyCatEx** [immich-app/immich#6624] [Feature Request]
### The feature

All the photos in the album are displayed directly. I hope the photos in the album can also be arranged like the timeline in the homepage.

It should be like this:
2024-01
photos in 2024-01

2023-12
photos in 2023-12

2022-11
photos in 2023-11

Not:
photos sorted simply by time

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #1689

**Reply by LovelyCatEx** [reply]
This is not duplicated. In #1689 it just noticed: " sort options".
My suggestion is: Photos in album could be sorted by datetime just like homepage.
It should be like this:
2024-01
photos in 2024-01

2023-12
photos in 2023-12

2022-11
photos in 2023-11

Not:
photos sorted simply by time

**Reply by LovelyCatEx** [reply]
I have edited this discussion. Will this new feature be considered in future versions?
